### PR TITLE
make urls in view configurable

### PR DESCRIPTION
**Pull request by shahmanash**

### CODE REVIEW

File: grails-app/conf/application.yml [hunk 1]
```diff
@@ -101,6 +101,7 @@ serverName: https://sds.ala.org.au
 security:
   cas:
     appServerName: https://sds.ala.org.au
+
```
Comment (charvolant) [inline grails-app/conf/application.yml]: Any reason why this is here?

File: grails-app/views/SDS/index.gsp [hunk 2]
```diff
@@ -10,15 +10,15 @@
...
 <ul>
     <li>Conservation Sensitivity, whereby sensitive species are denatured according to state rules. </li>
     <li>Pest Sensitivity. Species are passed through a set of rules based on categories as defined in the <b>Plant Biosecurity Sensitive Data Service</b></li>
 </ul>
-For more information see <a href="http://www.ala.org.au/faq/data-sensitivity/">Data Sensitivity</a>.
+For more information see <a href="${grailsApplication.config.sdsFAQ.url}">Data Sensitivity</a>.
 <h2>Testing</h2>
 <p>
-The Sensitive Data Service can be tested through the <a href="http://sandbox.ala.org.au">sandbox</a>. Simply upload your list of species and coordinates and the
+The Sensitive Data Service can be tested through the <a href="${grailsApplication.config.sandbox.url}">sandbox</a>. Simply upload your list of species and coordinates and the
```
Comment (charvolant) [inline grails-app/views/SDS/index.gsp:21]: default sandbox.url seems to be missing from application.yml

File: grails-app/views/SDS/index.gsp [hunk 3]
```diff
@@ -74,7 +74,7 @@ individual record pages of the sandbox will indicate whether or not sensitivity
     %{--<li> <a href="${request.contextPath}/sensitivity-zones.xml">sensitivity-zones.xml</a></li>--}%
 %{--</ul>--}%
 
-The species that make up the individual components of the SDS can be view via the <a href="http://lists.ala.org.au/public/speciesLists?isSDS=eq:true">list tool</a>.
+The species that make up the individual components of the SDS can be view via the <a href="${grailsApplication.config.list.tool.url}/public/speciesLists?isSDS=eq:true">list tool</a>.
```
Comment (charvolant) [inline grails-app/views/SDS/index.gsp:77]: lists.tool.url seems to be missing from application.yml

**Review by charvolant — CHANGES_REQUESTED** [review]
There's one odd line in application.yml which doesn't seem to make sense to me.
There doesn't seem to be a default sandbox.url or lists.tool.url in application.yml

File: grails-app/conf/application.yml (hunk 1 above)
Comment (shahmanash) [reply grails-app/conf/application.yml]: My bad , removed in 5b4ffac97b318e17cc792f018654c097bde50fb8

File: grails-app/views/SDS/index.gsp (hunk 2 above)
Comment (shahmanash) [reply grails-app/views/SDS/index.gsp:21]: Added in 5b4ffac97b318e17cc792f018654c097bde50fb8

File: grails-app/views/SDS/index.gsp (hunk 3 above)
Comment (shahmanash) [reply grails-app/views/SDS/index.gsp:77]: Added in 5b4ffac97b318e17cc792f018654c097bde50fb8

**Review by charvolant — APPROVED** [review]

**Pull request merged**